docs/guide/caching.md

```diff
@@ -125,25 +125,29 @@ extend: '@apostrophecms/piece-type',
 options: {
   cache: {
     api: {
+      // Specified in seconds
       maxAge: 3000,
       etags: true
     }
   }
 }
 ```
 
-_Note that for now, only single pieces benefit from this advanced caching system (`getOne` REST API piece responses)._
-
 You can also enable it for all piece types by enabling it for `@apostrophecms/piece-type`:
 
 ```javascript
 // modules/@apostrophecms/piece-type/index.js
 options: {
   cache: {
     api: {
+      // Specified in seconds
       maxAge: 3000,
       etags: true
     }
   }
 }
-```
\ No newline at end of file
+```
+
+Please note that even with the advanced cache invalidation system there will always be situations where caching allows a website visitor to see old content. Page content might differ based on an async component that fetches modified documents, or API calls to a third party service, or even the time of day. For this reason `maxAge` should always be set to a reasonable value to ensure the served content is never too old.
+
+For this reason content is never cached for logged-in users.
```

Comment:
Let's merge these at the end of the same paragraph and use the words "for this reason" only once:

"For this reason `maxAge` should always be set to a reasonable value to ensure the served content is never too old. Content is never cached for logged-in users."